src/mumble/AudioInput.cpp

```diff
@@ -815,13 +816,30 @@ void AudioInput::encodeAudioFrame() {
 
```

Comment:
Yeah I think this is sensible. After your suggested change, one thing to note is that tIdle.restart() is being called more aggressively now -- not just when `g.s.iaeIdleAction != Nothing`, as it was before. I had avoided messing with that, since I wasn't sure exactly how tIdle was being used everywhere in the code.
Upon looking at how tIdle is used, I think it should be fine, so I made the change.